ci-metrics/ci_listener.py

```diff
@@ -272,6 +276,79 @@ def handle_tests(self, key, value, retried=False):
                     (executor, next_slot)] = int(tester["failed"])
                 self.message_out["%s_time_spent_%s" %
                     (executor, next_slot)] = max(int(time_end - time_start), 0)
+                # Create some aggregated fields so that they
+                # don't have to be scripted in kibana
```

Comment:
again..  a lot of code duplication here that basically does the same thing.  Please move this into a helper function that takes an argument.  I would also get rid of the if else piece and simply use the following:

def message_key_addition(self, message_out_key, index_key):
    self.message_out[message_out_key] = self.message_out.get(message_out_key,0) + int(tester[index_key])

call it like this:
message_key_addition("executor_total_tests_exec", "executed")
message_key_addition("total_tests_exec", "executed")
....